src/NodeFinder/EmptyReturnNodeFinder.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\NodeFinder;
+
+use PhpParser\Node\Expr;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Return_;
+use Rector\Core\PhpParser\Node\BetterNodeFinder;
+
+final class EmptyReturnNodeFinder
+{
+    public function __construct(
+        private readonly BetterNodeFinder $betterNodeFinder
+    ) {
+    }
+
+    public function hasNoOrEmptyReturns(ClassMethod $classMethod): bool
+    {
+        /** @var Return_[] $returns */
+        $returns = $this->betterNodeFinder->findInstanceOf($classMethod, Return_::class);
```

Comment:
It should use findInstancesofInFunctionlikeScoped ref https://github.com/rectorphp/rector-src/blob/97a33aa69494836b44ce68e0803e2568233f3e06/src/PhpParser/Node/BetterNodeFinder.php#L447 so it can ignore return in inner function

Reply:
https://github.com/rectorphp/rector-symfony/pull/166

Reply:
:+1: